invenio_cli/cli/translations.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio module to ease the creation and management of applications."""
+
+import click
+
+from pathlib import Path
+
+from ..commands import TranslationsCommands
+from .utils import pass_cli_config, run_steps
+
+
+@click.group()
+def translations():
+    """Commands for translations management."""
+
+
+@translations.command()
+@click.option(
+    "--babel-ini", "-b", default="translations/babel.ini", help="Relative path to babel.ini (including filename)."
+)
+@pass_cli_config
+def extract(cli_config, babel_ini):
+    """Extract messages for i18n support (translations)."""
+    click.secho("Extracting messages...", fg="green")
+    steps = TranslationsCommands.extract(
```

Comment:
Pointing out issue: some _command_ classes are called as instance some as class. In some cases (e.g. services) the docker helper is needed (all good here). However, from the base class these classes inherit the need for a `cli_config`. Should this stay at `cli/` package level? and be passed as parameters if needed?